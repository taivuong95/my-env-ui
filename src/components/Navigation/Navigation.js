import React from 'react';
const Navigation = () => {
  return (
    <div className='nav-bar'>
      <div className='nav-bar__button--burger'>
        <span></span>
        <span></span>
        <span></span>
      </div>
      <div className='nav-bar__location'>
        <div className='nav-bar__location--app-name'>myENV</div>
        <div className='nav-bar__location--dropdown'>
          <span>Current Location</span>
          <span className='nav-bar__location--dropdown--icon'></span>
        </div>
      </div>
      <div className='nav-bar__notification'></div>
    </div>
  )
}

export default Navigation;