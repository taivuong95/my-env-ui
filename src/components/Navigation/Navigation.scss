.nav-bar{
    display: flex;
    justify-content: space-between;
    padding: 2rem;

    &__button--burger{
      display: inline-block;
      width: 3rem;
      height: 3.5rem;
      cursor: pointer;

      span{
        display: block;
        width: 100%;
        height: 3px;
        background-color: #fff;
        margin: 6px 0;
        transition: .4s;
      }
    }

    &__location{
      text-align: center;

      &--app-name{
        font-size: 1.6rem;
      }
      &--dropdown{
        display: flex;
        align-items: center;
        font-size: 1.6rem;
      }
      &--dropdown--icon{
        display: inline-block;
        width: 1.5rem;
        height: 1.5rem;
        background-size: contain;
        background-image: url(../../assets/img/down-arrow.svg);
        margin-left: 5px;
        margin-top: 5px;
      }
    }
    &__notification{
      width: 3rem;
      height: 3rem;
      background-size: contain;
      background-image: url(../../assets/img/bell.svg);
      cursor: pointer;
    }
  }

  
  