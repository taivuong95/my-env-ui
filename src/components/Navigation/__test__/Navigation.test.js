import React from 'react';
import Navigation from '../Navigation.js'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Navigation Component', () => {

    it("expect to render Navigation component", () => {
        const wrapper = shallow(<Navigation />);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})