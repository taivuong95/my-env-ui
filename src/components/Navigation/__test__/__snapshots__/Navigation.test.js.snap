// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component expect to render Navigation component 1`] = `
<div
  className="nav-bar"
>
  <div
    className="nav-bar__button--burger"
  >
    <span />
    <span />
    <span />
  </div>
  <div
    className="nav-bar__location"
  >
    <div
      className="nav-bar__location--app-name"
    >
      myENV
    </div>
    <div
      className="nav-bar__location--dropdown"
    >
      <span>
        Current Location
      </span>
      <span
        className="nav-bar__location--dropdown--icon"
      />
    </div>
  </div>
  <div
    className="nav-bar__notification"
  />
</div>
`;
